samples/msgext-ai-sentiment-analysis/nodejs/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-msgext-ai-sentiment-analysis-nodejs",
+    "source": "officeDev",
+    "title": "Sentiment analysis with messaging extension",
+    "shortDescription": "This sample shows a feature where user can get sentiment analysis from messaging extension.",
```

Comment:
This sample shows a feature where user can check analyze sentiments for messages posted in Teams chat using messaging extension  and Azure Open AI.

Reply:
fixed